assemble/Momentum_DG.F90

```diff
@@ -714,7 +757,11 @@ subroutine construct_momentum_dg(u, p, rho, x, &
             & inverse_mass=inverse_mass, &
             & inverse_masslump=inverse_masslump, &
             & mass=mass, subcycle_m=subcycle_m, subcycle_rhs=subcycle_rhs, &
-            & partial_stress=partial_stress)
+            & partial_stress=partial_stress, &
+            & have_les=have_les, smagorinsky_coefficient=smagorinsky_coefficient, &
```

Comment:
All other have_X logicals are set as module variables instead of passing it on, so let's stick with that convection shall we?